lib/src/tooltip_widget.dart

```diff
@@ -259,57 +274,106 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
+                                        children: <Widget>[
+                                          widget.title != null
+                                              ? Text(
+                                                  widget.title!,
+                                                  key: _key1,
+                                                  style:
+                                                      widget.titleTextStyle ??
+                                                          Theme.of(context)
+                                                              .textTheme
+                                                              .headline6!
+                                                              .merge(
+                                                                TextStyle(
+                                                                  color: widget
+                                                                      .textColor,
+                                                                ),
+                                                              ),
+                                                )
+                                              : SizedBox.shrink(),
+                                          widget.description != null
+                                              ? Text(
+                                                  widget.description!,
+                                                  key: _key2,
+                                                  style: widget.descTextStyle ??
+                                                      Theme.of(context)
+                                                          .textTheme
+                                                          .subtitle2!
+                                                          .merge(
+                                                            TextStyle(
+                                                              color: widget
+                                                                  .textColor,
+                                                            ),
+                                                          ),
+                                                )
+                                              : SizedBox.shrink(),
+                                        ],
+                                      ),
+                                    ),
+                                    if (widget.actions != null)
+                                      SizedBox(
+                                        height: widget
+                                                .spaceBetweenTitleAndButtons ??
+                                            5,
                                       ),
-                                    ],
-                                  ),
+                                    if (widget.actions != null)
+                                      LayoutBuilder(
```

Comment:
do we need LayoutBuilder here?

Reply:
we are calculating width based on render view that's why I used but without it it's working so removed.